src/applications/vaos/components/calendar/CalendarWidget.jsx

```diff
@@ -107,7 +107,9 @@ export default class CalendarWidget extends Component {
 
   isValid = date => {
     let isValid = true;
-    const { additionalOptions } = this.props.additionalOptions;
```

Comment:
Please verify this change.

Reply:
That definitely looks wrong, but I'd like @narin to take a look at it, since he would better understand the consequences of fixing it and enabling the code in here that's behind the conditional.

Reply:
Yes this is wrong, good catch.  Not too much of a concern since the entire `isValid` function will be removed in my other PR related to checkbox validation: https://github.com/department-of-veterans-affairs/vets-website/pull/11784.